core/web/schema/type/ocr.graphql

```diff
@@ -0,0 +1,10 @@
+type OCRKey {
```

Comment:
Would it be better to call this `OCRKeyBundle`? I believe that's what the REST API calls it. Perhaps this would also cascade to the query and mutation API's - `ocrKeyBundles` instead of `ocrKeys`, `createOCRKeyBundle` instead of `createOCRKeys`, etc. The payload would then be called `OCRKeyBundlesPayload`.